src/browser/tab/element/mod.rs

```diff
@@ -337,38 +341,65 @@ impl<'a> Element<'a> {
     }
 
     pub fn get_midpoint(&self) -> Fallible<Point> {
-        let return_object = self.parent.call_method(dom::methods::GetContentQuads {
-            node_id: None,
-            backend_node_id: Some(self.backend_node_id),
-            object_id: None,
-        })?;
-        let raw_quad = return_object.quads.first().unwrap();
-        let input_quad = ElementQuad::from_raw_points(&raw_quad);
-
-        Ok((input_quad.bottom_right + input_quad.top_left) / 2.0)
+        match self
+            .parent
+            .call_method(dom::methods::GetContentQuads {
+                node_id: None,
+                backend_node_id: Some(self.backend_node_id),
+                object_id: None,
+            })
+            .and_then(|quad| {
+                let raw_quad = quad.quads.first().unwrap();
+                let input_quad = ElementQuad::from_raw_points(&raw_quad);
+
+                Ok((input_quad.bottom_right + input_quad.top_left) / 2.0)
+            }) {
+            Ok(e) => return Ok(e),
+            Err(_) => {
+                let mut p = Point { x: 0.0, y: 0.0 };
+
+                util::Wait::with_sleep(Duration::from_secs(1)).run_until(|| {
+                    let r = self.call_js_fn(
+                        r#"
+                    function() {
+                        let v = document.getElementsByClassName("ShowAllChapters")[0];
```

Comment:
ShowAllChapters?

Reply:
Sorrr this is embarrasing, i forgot to remove this

Reply:
> Sorry this is embarrasing, i forgot to remove this

i was testing it with a element and forgot to remove it in my fork